lib/core.ps1

```diff
@@ -128,13 +128,35 @@ function setup_proxy() {
     }
 }
 
-function git_cmd {
+function Invoke-Git {
+    [CmdletBinding()]
+    [OutputType([String])]
+    param(
+        [String] $Path,
+        [PSObject[]] $ArgumentList
```

Comment:
```suggestion
        [String[]] $ArgumentList
```

And could we use `Invoke-ExternalCommand()` here? (The function cannot add EnvVars now, I'll submit PR to support it)

Reply:
I tried using `Invoke-ExternalCommand()` but the redirecting if `stdout` breaks the output of git. E.g.:

![image](https://user-images.githubusercontent.com/432127/186981571-26316a16-9c02-488d-8c2d-889732ff4c0e.png)